### PR TITLE
Add troubleshooting problem about telemetry issue in 7.11

### LINKED ISSUE #318: Quick 7.11 Docs note to support Endpoint Security Telemetry feature Agent side requirements
Hi.  @kevinlog mentions this to the Ingest Team, and we wanted to document it.  We will need to quickly confirm specifics to type up the docs.

https://github.com/elastic/kibana/pull/86277

This ticket merges a small change, about the Protections Team Telemetry feature, but I don't know off hand the larger meta ticket, sorry.  Kevin can you post and update?

Assigning Janeen and Dede to coordinate on which parts may go into one or both of Security side and Fleet / Agent side docs.

We know this:

if you upgrade an Agent from pre-7.11 the Telemetry feature will be on by default in the 7.11 Agent and it will attempt to write to the new index per the above issue, but it will fail (because the API key, as upgraded, will not have permissions).

Nothing else bad will happen, apart from errors in the logs, mind you.  but we expect users to notice and ask about it so we can be proactive.

The work arounds we have so far are this:
 - re-enroll the Agent to pick up the API key changes (which should be available automatically after upgrading, right Kevin?)
 - change the Kibana side policy for this Telemetry feature to turn it off so it won't keep logging error messages on Elastic Endpoint side as well as Elastic index side (likely 1 error per Elastic Endpoint per some time frame).  We don't know for sure what the back-off policy is on attempting this, so it could be a lot of noise in the Elastic and Endpoint side logs.  Kevin was going to log a ticket for improving that back-off strategy for this specific case (as needed).

### PR DESCRIPTION
**Pull request by dedemorton**
Closes #318

### CODE REVIEW

File: docs/en/ingest-management/troubleshooting.asciidoc [hunk 1]
```diff
@@ -178,4 +178,37 @@ If this happens, select **Force unenroll** from the *Actions* menu in the
...
 
 This will invalidate all API keys related to the agent and change the status to
-`inactive` so that the agent no longer appears in {fleet}. 
+`inactive` so that the agent no longer appears in {fleet}.
+
+[discrete]
+[[endpoint-unauthorized]]
+== API key is unauthorized to send telemetry to `.logs-endpoint.diagnostic.collection-*` indices
+
+Telemetry is turned on by default in {kib}. This helps us learn about the
```
Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting.asciidoc:187]: Not sure if this should say "in Kibana" or "in the Elastic Stack.

Comment (kevinlog) [reply docs/en/ingest-management/troubleshooting.asciidoc:187]: Say Elasticstack

**Review by kevinlog — APPROVED** [review]

File: docs/en/ingest-management/troubleshooting.asciidoc (hunk 1 above)
Comment (kevinlog) [reply docs/en/ingest-management/troubleshooting.asciidoc:187]: *Elastic Stack

Comment (dedemorton) [inline docs/en/ingest-management/troubleshooting.asciidoc:187]:
```suggestion
Telemetry is turned on by default in the {stack}. This helps us learn about the
```

**Pull request merged**